media-collection/public/media-automatic-session-handling.js

```diff
@@ -0,0 +1,267 @@
+const createVideoPlayer = videoPlayerId => {
```

Comment:
Can you add a comment to the top of these javascript files explaining the differences? i.e. for this one you are letting Web SDK handle adding the sessionID to the media events.

Reply:
yes, good idea, thank you